app/models/chargeback.rb

```diff
@@ -59,6 +59,10 @@ def self.build_results_for_report_chargeback(options)
         metric_rollup_record = metric_rollup_records.first
         rates_to_apply = rates.get(metric_rollup_record)
 
+        if metric_rollup_record.resource_type == Container.name && rates_to_apply.empty?
+          rates_to_apply = [ChargebackRate.find_by_description("Default Container Image Rate")]
+        end
+
```

Comment:
can  this be moved to RatesCache class if it is possible ?
and maybe `ChargebackRate.find_by(:description => 'Default Container Image Rate')`  (I think find_by_description is deprecated)